website/src/client/signin.tsx

```diff
@@ -2,19 +2,19 @@ import { Auth } from "aws-amplify"
 import React from "react"
 import { Button } from "src/components"
 import Link from "src/components/link"
-import { UserProvider, useCredentials } from "../auth"
+// import { UserProvider, useCredentials } from "../auth"
 
-const ClientSignIn = () => {
-  const creds = useCredentials()
-  if (creds) {
-    return <Button onClick={() => Auth.signOut()}>Sign Out</Button>
-  } else {
-    return <Link href="/signin">Sign In</Link>
-  }
-}
+// const ClientSignIn = () => {
```

Comment:
If you think these components are not needed anymore, you can delete the file.

Reply:
Aaand it's gone!